configure.ac

```diff
@@ -213,6 +213,7 @@ AC_CHECK_HEADERS([endian.h sys/endian.h])
 AC_CHECK_HEADERS([utmpx.h])
 AC_CHECK_HEADERS([termio.h])
 AC_CHECK_HEADERS([sys/uio.h])
+AC_CHECK_HEADERS([stdarg.h syslog.h])
```

Comment:
Since syslog.h and the syslog family of calls are in POSIX, we don't need an autoconf header check.  (We might need a define check for auth/authpriv if we decide to go that route.)

Can you put this under a configure option, though?